pkgs/development/python-modules/pytest-xdist/default.nix

```diff
@@ -1,30 +1,27 @@
-{ stdenv, fetchPypi, buildPythonPackage, execnet, pytest
+{ stdenv, fetchPypi, buildPythonPackage, execnet, pytest_6
```

Comment:
but you did make the changes in this file, right?

pytest == pytest 6 so don't change this

Reply:
>but you did make the changes in this file, right?

No.

Reply:
I did this one, pytest_xdist_2 only works with pytest_6. This was back like 2 months ago